rust/operator-binary/src/env_vars.rs

```diff
@@ -299,22 +299,34 @@ fn static_envs(airflow: &AirflowCluster) -> BTreeMap<String, EnvVar> {
 pub fn build_gitsync_statefulset_envs(
     rolegroup_config: &HashMap<PropertyNameKind, BTreeMap<String, String>>,
 ) -> Vec<EnvVar> {
-    let mut env = vec![];
+    let mut env: BTreeMap<String, EnvVar> = BTreeMap::new();
```

Comment:
The thinking here (a map instead of a vector) was to ensure that the envs are always written in the same order, to avoid restarts/new versions etc. (can't remember the exact details) that caused problems. I'm not sure if this only an issue for the templates (but it may be a good idea to do it for the statefulsets as well).

Reply:
_Order_ is `Vec`'s nickname :)